protected/views/site/help.php

```diff
@@ -1,362 +1,929 @@
+                            </li>
+                            <li>
+                                Refine search to Authors<br />
+                                <a href="http://gigadb.org/api/search?author=Wang%20Jun" target="_blank">http://gigadb.org/api/search?author=Wang%20Jun</a>
+                            </li>
+                            <li>
+                                Refine search to linked manuscript IDs<br />
+                                <a href="http://gigadb.org/api/search?manuscript=10.1371/journal.pone.0005795" target="_blank">http://gigadb.org/api/search?manuscript=10.1371/journal.pone.0005795</a>
+                            </li>
+                            <li>
+                                Refine search to dataset types<br />
+                                <a href="http://gigadb.org/api/search?datasettype=Genomic" target="_blank">http://gigadb.org/api/search?datasettype=Genomic</a>
+                            </li>
+                            <li>
+                                Refine search to project names<br />
+                                <a href="http://gigadb.org/api/search?project=Genome%2010K&amp;result=sample" target="_blank">http://gigadb.org/api/search?project=Genome%2010K&amp;result=sample</a>
+                            </li>
+                            <li>
+                                List all published dataset DOIs (listed in publication date order)<br />
+                                <a href="http://gigadb.org/api/list" target="_blank">http://gigadb.org/api/list</a>
+                            </li>
+                            <li>
+                                Dump the database<br />
+                                <a href="http://gigadb.org/api/dump" target="_blank">http://gigadb.org/api/dump</a>
+                            </li>
+                            <li>
+                                List all dataset DOI's published in a date range (results ordered by
+                                publication date)<br />
+                                <a href="http://gigadb.org/api/list?start_date=2018-01-01&amp;end_date=2018-01-30" target="_blank">http://gigadb.org/api/list?start_date=2018-01-01&amp;end_date=2018-01-30</a>
+                            </li>
+                        </ol>
+                    </section>
+
+                    <hr class="dashed">
+                    <section aria-labelledby="cliTitle" class="m-0">
+                    <h2 class="page-subtitle" id="cliTitle">Command line usage</h2>
+                    <p>You can also use the curl commands on the command line to retrieve metadata, e.g.:</p>
+                    <pre><code>curl <a href="http://gigadb.org/api/dataset?doi=100051" target="_blank" rel="noopener noreferrer">http://gigadb.org/api/dataset?doi=100051</a></code></pre>
```

Comment:
used `<pre>` and `<code>` elements for semantics and a better visual display of commands